CHANGES/3620.bugfix

```diff
@@ -0,0 +1 @@
+Allow installation with pulpcore/3.55.
```

Comment:
Should we somehow sync up this formulation across plugins?

Also: pulp_rpm seems to choose a new ad-hoc formulation each time, from the changelog:

- Features: Added pulpcore 3.40 compatibility.
- Features: Declares (and requires at least) pulpcore/3.25 compatibility.

Why is this suddenly a bugfix and can we become more consistent?

Reply:
I added a comment about this on the pulpcore 3.55 discourse channel as well: https://discourse.pulpproject.org/t/pulpcore-3-55-is-coming-soon/1259/3?u=quba42

Reply:
Discussion from rpm-team-mtg: previous such releases have been features because they relied-on functionality in the breaking-release, and so needed their lowerboud raised. We generally try to allow the widest possible installation/use of pulp_rpm. If you look at the tests here, the current PR works cleanly back to the current LB ofrpm/3.44.1 (see https://github.com/pulp/pulp_rpm/actions/runs/9573762412/job/26468187737#step:21:53). Given that this change can be backported to previous releases, we mark it as bugfix.

That being said - the desire for there to be some similarity/compatibility for changelog entries across plugins is a good one. Will change.

Reply:
Of course, what we're *really* declaring is "compatibility with pulpcore<3.70".